### PR TITLE
[SimCalorimetry/EcalElectronicsEmulation] Solve warning found by gcc 5.3.0.

### PR DESCRIPTION
**Pull request by ghost**
https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/slc6_amd64_gcc530/CMSSW_8_1_X_2016-03-02-1100/SimCalorimetry/EcalElectronicsEmulation
At compile time the class method hasn't evaluated the array parameter therefore evaluating the "sizeof" of the array parameter as the sizeof a pointer of that type.

### CODE REVIEW

**Comment by degano** [comment]
please test

**Comment by civanch** [comment]
+1

File: SimCalorimetry/EcalElectronicsEmulation/src/EcalSimRawData.cc [hunk 1]
```diff
@@ -481,7 +481,7 @@ void EcalSimRawData::getSrfs(const edm::Event& event,
   //EE
   edm::Handle<EESrFlagCollection> hEeSrFlags;
   event.getByLabel(srDigiProducer_, eeSrFlagCollection_, hEeSrFlags);
-  for(size_t i=0; i < sizeof(eeSrf)/sizeof(int); ((int*)eeSrf)[i++] = -1){};
+  for(size_t i=0; i < (nEndcaps*nScX*nScY); ((int*)eeSrf)[i++] = -1){};
```
Comment (smuzaffar) [inline SimCalorimetry/EcalElectronicsEmulation/src/EcalSimRawData.cc]: I think this multiplication operation is going to perform for every iteration ..... may be better to calculate it once outside the loop?

**Comment by davidlange6** [comment]
+1

**Pull request merged**